caterva2/services/sub.py

```diff
@@ -344,44 +345,34 @@ async def partial_download(abspath, path, slice_):
                 await download_chunk(path, schunk, n)
 
 
-@app.get('/api/download/{path:path}')
-async def download_data(path: str, slice_: str = None, download: bool = False):
+@app.get('/api/fetch/{path:path}')
```

Comment:
Not sure whether the names "fetch" and "download" convey the difference in meaning to the reader. It may be worth considering alternatives to "fetch" like "data", "dump" or even "slice". I'm afraid I don't have very good alternatives, though. :slightly_frowning_face:

Reply:
Yeah, naming things is always a funny game.